src/SprykerEco/Zed/AvalaraTax/Business/Calculator/AbstractCartItemAvalaraTaxCalculator.php

```diff
@@ -197,4 +198,22 @@ protected function convertToPercents(float $number): float
     {
         return $number * 100.0;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\CalculableObjectTransfer $calculableObjectTransfer
+     * @param \Generated\Shared\Transfer\AvalaraCreateTransactionResponseTransfer $avalaraCreateTransactionResponseTransfer
+     *
+     * @return void
+     */
+    protected function setAvalaraCreateTransactionResponseToOriginalQuote(
```

Comment:
maybe it's better to rename this method to `getAvalaraCreateTransactionResponseToOriginalQuote` and to return `?QuoteTransfer`?
It will make this method a bit more clear.